File: Java OOP/Homework05/models/Person.java
package models;

public abstract class Person  {
    protected int age;
    protected String surname;
    protected String name;
    protected String patronymic;
    protected String telephone;

    public Person(int age, String surname, String name, String patronymic, String telephone) {
        this.age = age;
        this.surname = surname;
        this.name = name;
        this.patronymic = patronymic;
        this.telephone = telephone;
    }

    public int getAge() {
        return this.age;
    }
    
    public String getSurname() {
        return this.surname;
    }
    
    public String getName() {
        return this.name;
    }
    
    public String getPatronymic() {
        return this.patronymic;
    }
    
    public String getTelephone() {
        return this.telephone;
    }

    public void setAge(int age) {
        this.age = age;
    }
    
    public void setSurname(String surname) {
        this.surname = surname;
    }
    
    public void setName(String name) {
        this.name = name;
    }
    
    public void setPatronymic(String patronymic) {
        this.patronymic = patronymic;
    }
    
    public void setTelephone(String telephone) {
        this.telephone = telephone;
    }
}

File: Java OOP/Homework07/models/Person.py
class Person:
    def __init__(self, age: int, surname: str, name: str, patronymic: str, telephone: str):
        self.age = age
        self.surname = surname
        self.name = name
        self.patronymic = patronymic
        self.telephone = telephone

    def get_age(self) -> int:
        return self.age
    
    def get_surname(self) -> str:
        return self.surname
    
    def get_name(self) -> str:
        return self.name
    
    def get_patronymic(self) -> str:
        return self.patronymic
    
    def get_telephone(self) -> str:
        return self.telephone

    def set_age(self, age: int):
        self.age = age
    
    def set_surname(self, surname: str):
        self.surname = surname
    
    def set_name(self, name: str):
        self.name = name
    
    def set_patronymic(self, patronymic: str):
        self.patronymic = patronymic
    
    def set_telephone(self, telephone: str):
        self.telephone = telephone
